hack/print-e2e-suite.sh

```diff
@@ -25,5 +25,6 @@ set -o pipefail
 DIR=$(cd $(dirname "${BASH_SOURCE}")/.. && pwd -P)
 
 $DIR/test/e2e/e2e.test \
-    -ginkgo.noColor \
-    -ginkgo.dryRun | sed "s|$DIR/|File: |g" | sed 's/•//g' | tail -n+5 | head -n-3
+    --ginkgo.no-color \
+    --ginkgo.v \
```

Comment:
Is this required?

Reply:
IIRC yes since it wouldn't display the list of the test files otherwise. See my message above where it only prints "Will run 423 of 423 specs"